src/server/services/wms/qgswmsparameters.cpp

```diff
@@ -758,6 +758,8 @@ namespace QgsWms
       f = Format::TEXT;
     else if ( fStr.startsWith( QLatin1String( "application/vnd.ogc.gml" ), Qt::CaseInsensitive ) )
       f = Format::GML;
+    else if ( fStr.startsWith( QLatin1String( "application/json" ), Qt::CaseInsensitive ) )
```

Comment:
It looks like the official media type for geojson is "application/geo+json". See https://tools.ietf.org/html/rfc7946. For compatiblity with GeoServer QGIS Server could support both "application/json" and "application/geo+json".

Reply:
@elemoine I added the `application/geo+json` format as well as a unit test.

Thanks for your input!